analyzer/src/funTest/assets/projects/synthetic/excludes-expected-result.yml

```diff
@@ -19,6 +19,10 @@ repository:
         comment: "Project not delivered."
       - path: "analyzer/src/funTest/assets/projects/synthetic/project-with-excludes/gradle/app1/build.gradle"
         type: "OTHER"
+        packages:
+        - id: "Maven:org.apache.commons:commons-text:1.1"
+          reason: "OTHER"
+          comment: "Undelivered dependency."
```

Comment:
Nit: We should say "Non-distributed" here (and also in other places like in line 19 above, which is not modified by this commit). Maybe add an accordingly cleanup-commit afterwards?